lib/cli/main.py

```
@@ -0,0 +1,647 @@
+
+    dest = user_blocks_dir / file.name
+    shutil.copy2(file, dest)
+    console.print(f"[green]✓[/green] Copied {file.name} to {user_blocks_dir}")
+
+    client = get_client()
+
+    # trigger server-side reload so the new block is registered
+    try:
+        client.reload_extensions()
+    except Exception:
+        pass  # server may be unreachable; validation below will report the issue
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)